### PR TITLE
✨ feat(theme): add notify

### PR DESCRIPTION
**Pull request by evpeople**

### CODE REVIEW

File: schemas/theme.json [hunk 1]
```diff
@@ -122,6 +122,18 @@
...
 		},
+		"notify": {
+			"type": "object",
+			"properties": {
+				"title_info": { "type": "string" },
+				"title_warn": { "type": "string" },
+				"title_error": { "type": "string" },
+				"icon_info": { "$ref": "#/$defs/style" },
+				"icon_warn": { "$ref": "#/$defs/style" },
+				"icon_error": { "$ref": "#/$defs/style" }
```
Comment (uncenter) [inline schemas/theme.json:133]: By the looks of it I think you mixed up the `title_` and `icon_` properties. The `title_` properties should use the `style` definitions and the `icon_` properties should be strings right?

**Review by uncenter — CHANGES_REQUESTED** [review]

File: schemas/theme.json (hunk 1 above)
Comment (evpeople) [reply schemas/theme.json:133]: I am not sure. I just copy them from sxyazi/yazi#717 this issue.

Comment (evpeople) [reply schemas/theme.json:133]: Oh I messed it up, it's indeed my fault.

Comment (uncenter) [reply schemas/theme.json:133]: Huh? You tell me!

**Comment by evpeople** [comment]
I made a new commit to fix it

**Comment by uncenter** [comment]
Thank you! I incorporated this into https://github.com/yazi-rs/schemas/commit/9ad10b60e877ee406b6e826a5d61950d771845e4 since there were some conflicts.

**Pull request closed without merging**